packages/components/src/components/navigation-item/navigation-item.ts

```diff
@@ -70,6 +71,8 @@ export default class SdNavigationItem extends SolidElement {
   /** Reflects HTML details element state and allows control from parent. Only used if `vertical` is true, no `href`is undefined, and `children` is defined. */
   @property({ type: Boolean, reflect: true }) open = false;
 
+  @property({ type: Boolean, reflect: true }) separated = false;
```

Comment:
Same feedback as above: for me this is not speaking enough. Alternatives like: separate-action would be speaking